library/joomla

```diff
@@ -1,49 +1,49 @@
-# this file is generated via https://github.com/joomla/docker-joomla/blob/65b178365a9294612ba364adb2e12c02df056430/generate-stackbrew-library.sh
+# this file is generated via https://github.com/joomla-docker/docker-joomla/blob/0c6aa57f33a4ac2f371c35467191424730b23078/generate-stackbrew-library.sh
 
 Maintainers: Michael Babker <michael.babker@joomla.org> (@mbabker)
```

Comment:
You may want to update your `generate-stackbrew-library.sh` with the new maintainer too.

Reply:
cc @HLeithner @zero-24

Reply:
Sorry did not noticed earlier here is the PR: https://github.com/joomla-docker/docker-joomla/pull/103